libImaging/Resample.c

```diff
@@ -107,7 +107,7 @@ ImagingPrecompute(int inSize, int outSize, struct filter *filterp,
     if ( ! kk)
         return 0;
 
-    xbounds = calloc(outSize * 2, sizeof(int));
+    xbounds = malloc(outSize * 2 * sizeof(int));
```

Comment:
I've added the [comment](https://github.com/python-pillow/Pillow/pull/1933/files#diff-b1e715e96fb950759e657d15a0b3f4f9R105) why we need calloc.